roles/enable_lmod/tasks/main.yaml

```diff
@@ -18,8 +18,8 @@
     - { path: 'cm-lmod-init.csh', regexp: 'LMOD.*$', replace: 'LMOD "1"'}
 
 - name: Put DefaultModules in place
-  copy:
-    src: DefaultModules.lua
+  template:
+    src: DefaultModules.lua.j2
     dest: "{{ item }}"
```

Comment:
Can you change this line to `"{{ item }}"/DefaultModules.lua` . The env var `LMOD_SYSTEM_DEFAULT_MODULES` refers to `/usr/share/modulefiles` for lua script defining default modules. It won't pick up anything that doesn't end. with a .lua extension. I tested it.

This will fix the error we are seeing at the login
```
The system default contains no modules
  (env var: LMOD_SYSTEM_DEFAULT_MODULES is empty)
  No changes in loaded modules
```

Reply:
Done in 8153567